cms/envs/common.py

```diff
@@ -1135,12 +1099,7 @@
 
 # Email
 TECH_SUPPORT_EMAIL = 'technical@example.com'
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-EMAIL_HOST = 'localhost'
-EMAIL_PORT = 25
-EMAIL_USE_TLS = False
-EMAIL_HOST_USER = ''
-EMAIL_HOST_PASSWORD = ''
+EMAIL_FILE_PATH = Derived(lambda settings: settings.DATA_DIR / "emails" / "studio")
```

Comment:
Testing this, I got:

```
     EMAIL_FILE_PATH = Derived(lambda settings: settings.DATA_DIR / "emails" / "lms")
                                                ~~~~~~~~~~~~~~~~~~^~~~~~~~~~
 TypeError: unsupported operand type(s) for /: 'str' and 'str'
```

Not entirely sure why your change triggered this, but anyway, I think the best fix is to explicitly construct a `Path` object before using `/` to join. Even if our default DATA_DIR is a Path instance, the operator (or Tutor) might override it to a plain str instance before the `Derived` body is evaluated.

```suggestion
EMAIL_FILE_PATH = Derived(lambda settings: path(settings.DATA_DIR) / "emails" / "studio")
```